core/controllers/profile.py

```diff
@@ -368,8 +372,18 @@ def get(self):
         if not constants.ENABLE_ACCOUNT_EXPORT:
             raise self.PageNotFoundException
 
-        user_data = takeout_service.export_data_for_user(self.user_id)
-        self.render_json(user_data)
+        user_data, user_images = takeout_service.export_data_for_user(
+            self.user_id)
+        memfile = python_utils.string_io()
```

Comment:
I'm not sure this is a standard term. Maybe use a more descriptive name perhaps?

Reply:
Updated. Let me know if this needs work.